core_backend/app/config.py

```diff
@@ -24,7 +24,7 @@
 # for all of its endpoints.
 LITELLM_MODEL_EMBEDDING = os.environ.get("LITELLM_MODEL_EMBEDDING", "openai/embeddings")
 LITELLM_MODEL_DEFAULT = os.environ.get("LITELLM_MODEL_DEFAULT", "openai/default")
-LITELLM_MODEL_SUMMARIZATION = os.environ.get(
+LITELLM_MODEL_GENERATION = os.environ.get(
     "LITELLM_MODEL_SUMMARIZATION", "openai/generate-response"
```

Comment:
Should we update the environment variable name as well?

Reply:
Omg yes.

Reply:
Fixed! Thank you for catching this!